pages/ksphere/kommander/1.1/release-notes/index.md

```diff
@@ -50,369 +50,109 @@ Template:
 
 -->
 
-### Kommander 1.1.0 RC 7 - July 16th 2020
+#### New Features
+
+##### Centralized Cost Monitoring
+
+Kubecost, running on Kommander, provides centralized cost monitoring for all managed clusters. This feature, installed by default in every Kommander cluster, provides a centralized view of Kubernetes resources used on all managed clusters. For more information go to [Centralized Cost Monitoring](https://docs.d2iq.com/ksphere/kommander/1.1.0/centralized-cost-monitoring/)
+
+##### D2-ify Managed Clusters
+
+When attaching non-Konvoy clusters, such as Amazon EKS, Azure AKS, Google GKE, and On-Premises Kubernetes clusters, Kommander will federate a subset of standard Konvoy Addons to enable SSO, AuthN/Z, observability, and cost monitoring. This includes addons: Prometheus Operator, Alert Manager, Karma, Elasticsearch, Kibana, and Fluentbit.
```

Comment:
Couldn't find a doc describing the addons we federate

Reply:
If we think this needs additional information, we can open a JIRA. We have to remember that release notes may not stick around forever, so making sure we talk about this stuff outside of release notes is usually necessary.

Reply:
yup i've asked @drewvanstone's team to write up a doc. We can link to it after